src/SapiEmitter.php

```diff
@@ -98,6 +98,13 @@ public function emit(ResponseInterface $response, bool $withoutBody = false): vo
             }
         }
 
+        /**
+         * Sends headers before the body.
+         * Makes a client possible to recognize the type of the body content if it is sent with a delay,
+         * for instance, for a streamed response.
+         */
+        flush();
```

Comment:
What about `flush()` in `emitBody()` method?

Reply:
Do you mean to move it there?
Logically it sends all headers to a client. Sending headers in the "emit body" makes me confusing.